### ISSUE TITLE
 Adding archive failed: archiveManager is not defined

### ISSUE THREAD

**Issue by tamashiisama** [buttercup/buttercup-browser-extension#152]
Hi,

I can't add a password archive from box.com (webdav) with buttercup-browser-extension.
I've tested with buttercup-desktop on windows 10 and it works perfectly.

### **System Info:**
Buttercup 1.9.0 Addon on Chrome
Chrome 67.0.3396.99 

### **Javascript error:**

```
setup-index.js:1 [info] Sending state update to background {type: "addArchive/setAdding", payload: true, actionSync: 20}
setup-index.js:1 [info] Sending state update to background {type: "app/setBusy", payload: "Adding existing archive...", actionSync: 21}
setup-index.js:1 [info] Making request to background for storing a new archive
setup-index.js:1 [info] Sending state update to background {type: "app/unsetBusy", actionSync: 22}
setup-index.js:1 Error: Adding archive failed: archiveManager is not defined
    at setup-index.js:1
    at sendResponseAndClearCallback (extensions::messaging:419)
    at messageListener (extensions::messaging:451)
    at EventImpl.dispatchToListener (extensions::event_bindings:403)
    at Event.publicClassPrototype.(anonymous function) [as dispatchToListener] (extensions::utils:138:26)
    at EventImpl.dispatch_ (extensions::event_bindings:387)
    at EventImpl.dispatch (extensions::event_bindings:409)
    at Event.publicClassPrototype.(anonymous function) [as dispatch] (extensions::utils:138:26)
    at dispatchOnMessage (extensions::messaging:392)
(anonymous) @ setup-index.js:1
Promise.catch (async)
(anonymous) @ setup-index.js:1
(anonymous) @ setup-index.js:1
(anonymous) @ setup-vendor.js:1
value @ setup-index.js:1
onClick @ setup-index.js:1
(anonymous) @ setup-buttercup.js:43
invokeGuardedCallback @ setup-buttercup.js:43
invokeGuardedCallbackAndCatchFirstError @ setup-buttercup.js:43
Ga @ setup-buttercup.js:43
Ka @ setup-buttercup.js:43
Ma @ setup-buttercup.js:43
Ia @ setup-buttercup.js:43
Pa @ setup-buttercup.js:43
Qa @ setup-buttercup.js:43
Ud @ setup-buttercup.js:43
batchedUpdates @ setup-buttercup.js:43
ec @ setup-buttercup.js:43
Yd @ setup-buttercup.js:43
interactiveUpdates @ setup-buttercup.js:43
Xd @ setup-buttercup.js:43
setup-index.js:1 [info] Sending state update to background {type: "addArchive/setAdding", payload: false, actionSync: 23}
```

**Comment by perry-mitchell** [comment]
Hi @tamashiisama - You're right, there's a bug on [this line](https://github.com/buttercup/buttercup-browser-extension/blob/346f7cf7b92dca900a369644003891a770a51157/source/background/library/archives.js#L157). I'll try to get it patched right away.

**Comment by perry-mitchell** [comment]
I've released v1.9.1 of the Chrome extension with the fix. Sorry for the inconvenience!

**Comment by tamashiisama** [comment]
Thank you :wink:
